grid.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <title>Noughts and Crosses</title>
+    <style>
+        .row div {
+            padding: 10px;
+            border: 1px solid black;
+            height:120px;
+            width: 120px;
+            float: left;
+        }
+        .row {
+            clear: both;
+        }
+    </style>
+    </head>
+<body>
+    <div>
+        <div class="row">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <div class="row">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <div class="row">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+    </div>
```

Comment:
Here's what I had here to go with my styles above. Note that the row class isn't specifically styled above. That would be considered bad practice in most places but I have left it in for now to clearly show intention.


    <div class="grid">
        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>

        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>

        <div class="row">
            <div class="box"></div>
            <div class="box"></div>
            <div class="box"></div>
        </div>
    </div>

Reply:
done